src/excel_adapter/internals/excel_retriever.go

```diff
@@ -9,20 +9,24 @@ import (
 	trace "github.com/rs/zerolog/log"
 	"google.golang.org/api/drive/v3"
 	"google.golang.org/api/option"
+
+	_ "embed"
 )
 
 const SHEETS_MIME_TYPE = "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet"
 
 type DownloadConfig struct {
-	Id          string
-	Path        string
-	Name        string
-	Credentials string
+	Id   string
+	Path string
+	Name string
 }
 
+//go:embed secret.json
+var api_key []byte
```

Comment:
don't use snake case for variables
```suggestion
var apiKey []byte
```